### PR TITLE
[new release] parsite (0.1)

### PR DESCRIPTION
**Pull request by faber-1**
A micro library for simple parsing combinators

- Project page: <a href="https://github.com/faber-1/parsite">https://github.com/faber-1/parsite</a>
- Documentation: <a href="https://github.com/faber-1/parsite">https://github.com/faber-1/parsite</a>

##### CHANGES:

Initial opam release!

### CODE REVIEW

**Comment by mseri** [comment]
You re redefining the result type in your library: it already exists in the standard library, including the monadic combinators. This will make the library likely incompatible or cumbersome to use with most other ocaml libraries

**Comment by mseri** [comment]
The dune file does not include a `public_name` so nothing will be installed by this package

File: packages/parsite/parsite.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```
Comment (mseri) [inline packages/parsite/parsite.0.1/opam:32]:
```suggestion
```
The tests are empty, this will avoid wasting computational power in the ci

**Comment by faber-1** [comment]
thanks! will go and make a few changes

**Pull request closed without merging**